app/en/home/arcade-cli/page.mdx

```diff
@@ -87,227 +95,287 @@ You can learn more about any of the commands by running `arcade <command> --help
+│ --show-packages                              Show loaded packages during discovery │
+│ --reload                                     Enable auto-reload on code changes    │
+│                                              (HTTP mode only)                      │
+│ --debug                                      Enable debug mode with verbose        │
+│                                              logging                               │
+│ --otel-enable                                Send logs to OpenTelemetry            │
+│ --env-file                          TEXT     Path to environment file              │
+│                                              [default: None]                       │
+│ --name                              TEXT     Server name                           │
+│                                              [default: None]                       │
+│ --version                           TEXT     Server version                        │
+│                                              [default: None]                       │
+│ --cwd                               TEXT     Working directory to run from         │
+│                                              [default: None]                       │
+│ --help                      -h               Show this message and exit.           │
+╰────────────────────────────────────────────────────────────────────────────────────╯
 ```
 
 ## `arcade deploy`
 
 ```bash
  Usage: arcade deploy [OPTIONS]
 
- Deploy toolkits to Arcade Cloud
-
-╭─ Options ───────────────────────────────────────────────────────────────────────────╮
-│ --deployment-file  -d      TEXT     The deployment file to deploy.                  │
-│                                     [default: worker.toml]                          │
-│ --host             -h      TEXT     The Arcade Engine host to register the worker   │
-│                                     to.                                             │
-│                                     [default: api.arcade.dev]                       │
-│ --port             -p      INTEGER  The port of the Arcade Engine host.             │
-│                                     [default: None]                                 │
-│ --tls                               Whether to force TLS for the connection to the  │
-│                                     Arcade Engine. If not specified, the connection │
-│                                     will use TLS if the engine URL uses a 'https'   │
-│                                     scheme.                                         │
-│ --no-tls                            Whether to disable TLS for the connection to    │
-│                                     the Arcade Engine.                              │
-│ --help                              Show this message and exit.                     │
-╰─────────────────────────────────────────────────────────────────────────────────────╯
+ Deploy MCP servers to Arcade
```

Comment:
This is a moving target, sorry. But the new help menu for `arcade deploy` looks like the following now:
<img width="3318" height="594" alt="image" src="https://github.com/user-attachments/assets/e977f05b-c41d-44e8-b970-fbd7fd92d8c2" />